content/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/classifying-your-repository-with-topics.md

```diff
@@ -47,3 +47,12 @@ You can search for repositories that are associated with a particular topic. For
   ![Form to enter topics](/assets/images/help/repository/add-topic-form.png)
 4. After you've finished adding topics, click **Save changes**.
   !["Save changes" button in "Edit repository details"](/assets/images/help/repository/edit-repository-details-save-changes-button.png)
+  
+## Limitations and quotas
```

Comment:
Instead of having an entirely new section, let's just bump this information up to the bottom of the "About topics" section.